docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -1512,14 +1512,14 @@ When a config client starts, it binds to the Config Server (through the `spring.
 The net result of this behavior is that all client applications that want to consume the Config Server need a `bootstrap.yml` (or an environment variable) with the server address set in `spring.cloud.config.uri` (it defaults to "http://localhost:8888").
 
 [[discovery-first-bootstrap]]
-==== Discovery First Bootstrap
+==== Discovery First Lookup
 
-If you use a `DiscoveryClient` implementation, such as Spring Cloud Netflix and Eureka Service Discovery or Spring Cloud Consul, you can have the Config Server register with the Discovery Service.
+WARNING: Unless you are using <<config-first-bootstrap, config first bootstrap>>, you will need to have a `spring.config.import` property in your configuration properties.
+For example, `spring.config.import=optional:configserver:`.  This functionality will be changed in a future release.
```

Comment:
Remove 'This functionality will be changed in a future release.' An import will always be required. No need to comment on future enhancements.